src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Per your jira ticket, you don't need a stories file for this component: https://tripleten-apiary.atlassian.net/browse/EES-15 
<img width="817" alt="image" src="https://github.com/user-attachments/assets/016ba6f7-c35f-4e49-84e9-7106fa6b70b0">

Reply:
Hey @XDRO so there was a sync where @tomershvueli did mention that there were some tickets that did not include a description for adding a story for a component. I think he told me to create one for the checkbox and textarea but hopefully he can give confirmation about that here.